pages/docs/dev-docs/release-strategy.md

```diff
@@ -38,6 +38,19 @@ permalink: dev-docs-release-strategy.html
 5. Make sure everything is working
 6. Release R
 
+## Release schedule
+
+The core library repository follows semantic versioning.
+
+We group together breaking changes and release a new breaking version as needed, but not more frequently than once every two years (aiming for longer). This aims to give time and motivation to users to adopt new versions, and spreads the effort of preparing a release over a longer period of time. There is no fixed schedule for major releases, and we take time to ensure that no further breaking changes are needed soon after.
```

Comment:
Do we already have a section where we explain the general workflow for a breaking release? This is not directly related to the schedule but I think it would be helpful to at least point to this workflow from here. 

Some points that are important in my opinion:

* What we did last time: At some point there was a `v3_develop` branch in parallel to the `develop` branch. At some later point we decided to merge `v3_develop` into `develop`. This closed the door for any v2 releases and forced everybody to integrate new features directly with v3.
* I think it would also be helpful to point to the milestone that we usually create in order to cluster breaking releases way before the breaking release is scheduled (see https://github.com/precice/precice/milestone/20).

Reply:
I agree. How about moving these points into an issue or other PR?

Reply:
https://github.com/precice/precice.github.io/issues/477